apollo-ios/Sources/ApolloSQLite/SQLiteDatabase.swift

```diff
@@ -23,7 +23,13 @@ public protocol SQLiteDatabase {
   func deleteRecords(matching pattern: CacheKey) throws
   
   func clearDatabase(shouldVacuumOnClear: Bool) throws
-  
+
+  func setJournalMode<T: RawRepresentable>(mode: T) throws where T.RawValue == String
+
+}
+
+public extension SQLiteDatabase {
+  func setJournalMode<T>(mode: T) throws where T : RawRepresentable, T.RawValue == String { }
```

Comment:
This extension ensures it's not a breaking change for projects that have written their own `SQLiteDatabase` implementation.